.github/workflows/ci.yml

```diff
@@ -27,4 +27,5 @@ jobs:
         with:
           ruby-version: ${{ matrix.ruby }}
           bundler-cache: true
+          bundler: latest
```

Comment:
🗒️ This fixes a warning that would be emitted when parsing our `test/fixtures/Gemfile.lock` on older versions of Ruby, due to the fact the lock file was generated by a more recent version of Bundler.